index.js

```diff
@@ -173,13 +274,13 @@ module.exports.toASCII = (domainName, useSTD3, processingOption, verifyDnsLength
   });
 
   if (verifyDnsLength) {
-    const total = labels.slice(0, labels.length - 1).join(".").length;
-    if (total.length > 253 || total.length === 0) {
```

Comment:
What'd the root label be in this case? Would it be a TLD, or rather a empty label as in the DNS root zone?